docs/java-rest/usage.asciidoc

```diff
@@ -129,14 +180,18 @@ https://hc.apache.org/httpcomponents-core-ga/httpcore/apidocs/org/apache/http/Ht
  object
 
 When performing a request, an exception is thrown (or received as an argument
- in `ResponseListener#onSucces(Exception)` in the following scenarios:
+ in `ResponseListener#onFailure(Exception)` in the following scenarios:
 
 `IOException`:: communication problem (e.g. SocketTimeoutException etc.)
 `ResponseException`:: a response was returned, but its status code indicated
-an error (either `4xx` or `5xx`). A `ResponseException` originates from a valid
+an error (not `2xx`). A `ResponseException` originates from a valid
```

Comment:
It's `not 2xx` as it currently exists:

```java
return statusCode < 300 || (HttpHead.METHOD_NAME.equals(method) && statusCode == 404);
```

That honestly makes sense because it also excludes any possibility of `3xx`.